pkgs/top-level/all-packages.nix

```diff
@@ -17746,6 +17746,12 @@ in
 
     systemtap = callPackage ../development/tools/profiling/systemtap { };
 
+    system76-dkms = callPackage ../os-specific/linux/system76-dkms { };
```

Comment:
Also this is not using the `dkms` backend to compile against the linux kernel, we should probably stick with upstream in terms of naming convention.

Reply:
Done.  I've also added "-module" to the packages' `name` attributes, since `system76` is a bit vague for a package name.